link/feature-link-form/LinkForm.tsx

```diff
@@ -0,0 +1,67 @@
+import styles from './LinkForm.module.scss';
+import classNames from 'classnames/bind';
+import { useGetFolders } from '@/folder/data-access-folder';
+import { AddLinkModal } from '@/link/ui-add-link-modal';
+import { LinkForm as UiLinkForm } from '@/link/ui-link-form';
+import { ChangeEvent, KeyboardEventHandler, useState } from 'react';
+import { useIntersectionObserver } from '@/sharing/util/useIntersectionObserver';
+
+const cx = classNames.bind(styles);
+
+type LinkFormProps = {
+  hideFixedLinkForm?: boolean;
+};
+
+export const LinkForm = ({ hideFixedLinkForm = false }: LinkFormProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { data: folders } = useGetFolders();
+  const [selectedFolderId, setSelectedFolderId] = useState<number | null>(null);
```

Comment:
ㅎㅎㅎ 상희님 코드에서는 유독 state 초기값을 null로 넣어주는 경우가 많이 보이네요.

이렇게 하면 selectedFolderId를 쓸때마다 null 체크를 부수적으로 해줘야할텐데,
숫자 0으로 초기값넣어주고 number타입만 올수있게해주면 부수적인 처리가 따로 필요없어져서 편할거예요.
배열의 경우는 빈배열, string의 경우에는 빈문자열을 초기값으로 써주시면 됩니다.

```suggestion
  const [selectedFolderId, setSelectedFolderId] = useState<number>(0);
```

한번 시도해보세요 👍